src/lib/core/styles/tokens/field/_tokens.scss

```diff
@@ -0,0 +1,135 @@
+@use 'sass:map';
+@use '../../animation';
+@use '../../border';
+@use '../../elevation';
+@use '../../shape';
+@use '../../theme';
+@use '../../utils';
+
+$tokens: (
+  display: utils.module-val(field, display, block),
```

Comment:
I think we can get rid of this `display` token as it's likely more of an internal layout style.